README.md

```diff
@@ -1,6 +1,102 @@
-# envaridator
+# Envaridator
 
-Envaridator is a tiny environment variable validation library. It's main purpose is
-parsing, registering and validating environment variables.
+Envaridator is a really small environment variable validation library.
 
-Check `examples/` for example usage.
\ No newline at end of file
+## Install
+
+Installing the package can be achieved by:
+
+```
+yarn add envaridator
```

Comment:
We should split this in two different sections, i.e.
With yarn:
```
yarn add envaridator
```
With npm:
```
npm install --save envaridator
```

Reply:
done